### ISSUE TITLE
Search behavior changed

### ISSUE THREAD

**Issue by henriman** [markusfisch/PieLauncher#106]
It seems that the behavior of the search in the app list has changed. Now, after entering a character, all apps that contain that character are listed, _in alphabetical order_ (and not starting with the apps that start with that character). E.g., if I enter "M", Bandcamp appears before Maps. 

If this change was intentional, I would appreciate the option to change the behavior back in the settings. (Maybe I am also mistaken and this is how the search always worked, I just noticed accidentally opening the wrong app for some character combinations that I had already internalized; in that case, this would be a feature request.)

(P.S.: Thank you for developing this awesome launcher!)

**Comment by markusfisch** [comment]
Hi, and thanks for using Pie Launcher!

There was a change regarding how the search works indeed! But it was a while back already. Fortunately, you can choose the behaviour you want in the settings of the app now. Look for the item "App search" in the "App Drawer" category (probably near the bottom of the screen). There, you can choose "Name starts with search term".

**Comment by markusfisch** [comment]
Good point! And similar to https://github.com/markusfisch/PieLauncher/issues/108

I'm about to try this!
